WordPress/Classes/ViewRelated/Blog/SiteSettingsViewController.m

```diff
@@ -993,15 +997,9 @@ - (BOOL)savingWritingDefaultsIsAvailable
 
 - (IBAction)cancel:(id)sender
```

Comment:
Can we change this method name to "dismiss" or something to better reflect what it does?  (kind of following the logic behind the button name change).